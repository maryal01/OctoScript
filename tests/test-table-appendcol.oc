table<int, string, float> t = read("tests/sampleTable.csv", ["int", "string", "float"], ",");

list<bool> b = (true, true, false);

t.append_col(b);

write("test-table.appendcol.out", t, ",");