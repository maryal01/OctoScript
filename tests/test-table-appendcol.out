1, "a", 1.0, true
2, "b", 2.0, true
3, "c", 3.0, false
